packages/react-components/react-avatar/src/components/AvatarGroupOverflow/AvatarGroupOverflow.test.tsx

```diff
@@ -1,18 +1,63 @@
 import * as React from 'react';
-import { render } from '@testing-library/react';
+import { AvatarGroupItem } from '../AvatarGroupItem/AvatarGroupItem';
 import { AvatarGroupOverflow } from './AvatarGroupOverflow';
+import { avatarGroupOverflowClassNames } from './useAvatarGroupOverflowStyles';
 import { isConformant } from '../../common/isConformant';
+import { RenderResult } from '@testing-library/react';
+
+// testing-library's queryByRole function doesn't look inside portals
+function queryByRoleDialog(result: RenderResult) {
```

Comment:
you could just use `screen` directly from `@testing-library`

https://testing-library.com/docs/queries/about/#screen